msteams-platform/toolkit/register-teams-app-microsoft-entra-id.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Register Teams app in Microsoft Entra ID
+author: surbhigupta
+description: Registering your Teams app in Microsoft Entra ID helps you access Azure resources and enable Azure-related functions.
+ms.topic: how-to
+ms.author: surbhigupta
+ms.localizationpriority: medium
+---
+# Register Teams app in Microsoft Entra ID
+
+Registering your Teams app in Microsoft Entra ID allows the app to access Azure resources and enable Azure-related functions such as single sign-on (SSO) or multitenancy.
+
+## Before you register your app
+
+It's helpful if you learn about the configuration for registering your app in Microsoft Entra ID beforehand. Ensure that you're prepared to configure the following details prior to registering your app:
```

Comment:
```suggestion
Prior to registering your app in Microsoft Entra ID, it's helpful to learn about configuration to ensure that you're prepared to configure the following details:
```
Do you have a link we can provide the user for further reading?

Reply:
Good catch! I have added a link.